src/silvimetric/bounds.py

```diff
@@ -1,16 +1,55 @@
 import json
+import ast
 
-class Bounds(object):
+class Bounds(dict): #for JSON serializing
     def __init__(self, minx: float, miny: float, maxx: float, maxy: float):
         self.minx = float(minx)
         self.miny = float(miny)
         self.maxx = float(maxx)
         self.maxy = float(maxy)
 
+    def __eq__(self, other):
+        return  other.minx == self.minx and \
+                other.miny == self.miny and \
+                other.maxx == self.maxx and \
+                other.maxy == self.maxy
+    def __neq__(self, other):
```

Comment:
I think this needs to be `__ne__()` to make the warning go away

Reply:
You'll note I subclassed `dict` there to get free json serialization. 🤷 if that's the "right" thing to do but it was expedient.